lib/modules/commentNavigator.js

```diff
@@ -89,7 +88,7 @@ modules['commentNavigator'] = {
 						case 'images':
 							thisEle.setAttribute('title', 'Navigate through comments with images');
 							break;
-						case 'images':
+						case 'videos':
```

Comment:
I'm on Windows, but I just tried building with makelinks and it also doesn't work (but I get "cannot connect to extension", rather than it just not updating).

You shouldn't have issues with symlinks if you use gulp or grunt to build though, right? (both work for me)